pom.xml

```diff
@@ -1870,6 +1880,7 @@ under the License.
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.8.0</version> 
           <configuration>
+            <source>${targetJdk}</source>
```

Comment:
This (and the changes above) is not needed: with `<release>` set, the `maven-compiler-plugin` will take care of adding the missing dependencies when needed - see https://winterbe.com/posts/2018/08/29/migrate-maven-projects-to-java-11-jigsaw/#configure-plugins-for-java-11 for example

Reply:
I am simply unable to build the project from master. A simple `mvn package` leads to:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:1.4.1:enforce (enforce-maven) on project syncope: 
Execution enforce-maven of goal org.apache.maven.plugins:maven-enforcer-plugin:1.4.1:enforce failed: An API incompatibility 
was encountered while executing org.apache.maven.plugins:maven-enforcer-plugin:1.4.1:enforce: java.lang.ExceptionInInitializerError: null
```

...which is probably the result of me using Maven 3.6.0. There is no Maven wrapper that I can use here unfortunately, and I am not able to downgrade to Maven 3.5. It is possibly also related to the version of maven-enforcer plugin as well. (There is a related JIRA here as well)

Furthermore, I am using Intellij IDEA. Without these changes, the IDE is unable to build the syncope project and consistently picks up JDK 1.7 for the build from the Maven configuration. If you're seeing the build pass, that's great. It may be IDEA related of course, but I won't be able to move forward with more contributions without these changes.

Reply:
> which is probably the result of me using Maven 3.6.0

That's odd, I am using Maven 3.6.0 with no issues:
```
$ mvn -version
Apache Maven 3.6.0 (97c98ec64a1fdfee7767ce5ffb20918da4f719f3; 2018-10-24T20:41:47+02:00)
Maven home: /opt/apache-maven-3.6.0
Java version: 11.0.2, vendor: Azul Systems, Inc., runtime: /usr/lib/jvm/zulu-11-amd64
Default locale: it_IT, platform encoding: UTF-8
OS name: "linux", version: "4.15.0-46-generic", arch: "amd64", family: "unix"
```

and same does Jenkins - check [latest job](https://builds.apache.org/view/S-Z/view/Syncope/job/Syncope-master-deploy/3471/consoleFull):
```
[Syncope-master-deploy] $ /home/jenkins/tools/java/latest11/bin/java -Xmx2048M -Xss128M -XX:+CMSClassUnloadingEnabled -XX:+TieredCompilation -XX:TieredStopAtLevel=1 -cp /home/jenkins/jenkins-slave/maven35-agent.jar:/home/jenkins/tools/maven/apache-maven-3.6.0/boot/plexus-classworlds-2.5.2.jar:/home/jenkins/tools/maven/apache-maven-3.6.0/conf/logging jenkins.maven3.agent.Maven35Main /home/jenkins/tools/maven/apache-maven-3.6.0 /home/jenkins/jenkins-slave/remoting.jar /home/jenkins/jenkins-slave/maven35-interceptor.jar /home/jenkins/jenkins-slave/maven3-interceptor-commons.jar 39659
<===[JENKINS REMOTING CAPACITY]===>channel started
```

> Furthermore, I am using Intellij IDEA. Without these changes, the IDE is unable to build the syncope project and consistently picks up JDK 1.7 for the build from the Maven configuration. If you're seeing the build pass, that's great. It may be IDEA related of course, but I won't be able to move forward with more contributions without these changes.

Sorry to hear that, but since your troubles are due to your IDE, either change your IDE - I am using Netbeans without any trouble - or keep them private.